AUTHORS.txt

```diff
@@ -2,7 +2,7 @@ The Spyder Doc Contributors are composed of:
 
 * Pierre Raybaut <pierre.raybaut@gmail.com> (Original Spyder author).
 * Carlos Cordoba <ccordoba12@gmail.com> (Current Spyder maintainer).
-* C.A.M. Gerlach <CAM.Gerlach@Gerlach.CAM> (Current docs maintainer).
+* C.A.M. Gerlach <CAM.Gerlach@Gerlach.CAM> (Former docs maintainer).
```

Comment:
```suggestion
```

Reply:
Ultimately @ccordoba12 's and the current maintainer's decision; I have no objection per say but I would suggest not making a habit of listing every former maintainer or it gets long and confusing.

Reply:
But if you want it there I'm not going to argue, its not a big deal.

Reply:
@CAM-Gerlach, ok, if you prefer not to be listed, I'm fine with that.

@jitseniesen, please apply CAM's suggestion so I can merge this one.

Reply:
I have no preference, it just sets the precedent of listing every former maintainer. Up to you.